cron/pfl

```diff
@@ -1,2 +1,8 @@
 #!/bin/sh
+
+if [ -e /bin/systemctl ] && /bin/systemctl is-enabled --quiet pfl.timer; then
+	# Do not run pfl via cron if the respective systemd timer is enabled.
+	exit
```

Comment:
I think `exit 0` would be nicer because it will save `man 1 exit` just to be sure. Just my two cents.